README.md

```diff
@@ -60,6 +74,11 @@ If you do not have a token for the operator, you can generate a token and save i
 kubectl create secret generic onepassword-token --from-literal=token=$(op create connect token <server> op-k8s-operator --vault <vault>)
 ```
 
+**Build Operator docker image**
+```
+make docker-build
+```
+
```

Comment:
You won't need this if the image is pulled from DockerHub. We should encourage the user to use the image 😄

Reply:
Ok 😁